src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. Error handling: The divide method correctly throws an IllegalArgumentException when attempting to divide by zero. This is a good practice. Consider adding a comment to explain this behavior near the throw statement.